deploy/README.md

```diff
@@ -1,31 +1,180 @@
 # Deploying the Arrebol Service
 
-The Arrebol service has two main components, the Arrebol Server and the Arrebol Worker. This document describes how to install the Arrebol server component. Before installing the Arrebol Server, please install the [Worker](../worker/deploy).
+This document provides an easy way to deploy a Arrebol Service. The Arrebol has two main components, the **Arrebol Server** and the **Worker Node**. Then, before installing the Arrebol Server, please deploy a [Worker Node](../worker/deploy).
 
-## Requirements
+## Overview
 
-The Docker service can be deployed as a docker container based on the from the [Arrebol docker image](https://hub.docker.com/repository/docker/ufcglsd/arrebol). To be able to run the Arrebol container, install the Docker Engine. To install it, in an apt-based environmnet, checkout this repository and run the `setup.sh` script as follow:
+The Arrebol Service manager the job execution in the worker nodes and monitor the job state. It mean that Arrebol uses the Worker Node resources to run the jobs.
+
+## Infrastruture
+
+You just need a machine for Arrebol Service.
+
+### Requirements
+* vCPU: 2
+* RAM: 2GB
+* FREE DISK SPACE: 10GB
+* OS: Ubuntu 16 or higher 
+
+### Security Group
+* Custom TCP Rule to allow ingress in the port **8080** to access **Arrebol API**
+* Custom TCP Rule to allow ingress in the port **5432** to access **Postgresql DB** (Optional)
+* Custom TCP Rule to allow ingress in the port **15432** to access **Pgadmin** (Optional)
+
+## Setup
+
+### 1.Download the repository
+
+Log in the arrebol host and follow the instructions.
+
+To download thre repository is need the git installed. Case it was not installed, run theses commands to install:
+```bash
+sudo apt update
+sudo apt install -y git
+```
+Run the below commands to download the arrebol repository:
+```bash
+git clone -b feature/remote-worker https://github.com/ufcg-lsd/arrebol.git
+```
+
+### 2.Install dependencies
+
+Log in the coordination host and run the following commands to install dependencies.
 
   ```
+  cd arrebol/deploy
   sudo bash setup.sh
   ```
 
-## Configuration
+### 3. Fill configuration files
+
+Go to the _deploy/config_ directory  inside _arrebol_:
+```bash
+cd arrebol/deploy/config
```

Comment:
if the user has done the previous step this cd command won't work, because he is inside arrebol/deploy at this time. Maybe you want to put the absolute path here.

Reply:
Done.